static/app/views/dashboards/widgetCard/autoSizedText.tsx

```diff
@@ -0,0 +1,177 @@
+import {useLayoutEffect, useRef, useState, useTransition} from 'react';
+import styled from '@emotion/styled';
+import {useResizeObserver} from '@react-aria/utils';
+import debounce from 'lodash/debounce';
+
+interface Props {
+  children: React.ReactNode;
+  maxFontSize: number;
+  minFontSize: number;
+  calculationCountLimit?: number;
```

Comment:
As a developer, how do I know what to set these to? Can this be simplified to only require children so that folks can only worry about the min container width and have the text autosize to it?